ui/imports/shared/popups/CommunityIntroDialog.qml

```diff
@@ -55,6 +35,8 @@ StatusDialog {
                 type: root.isInvitationPending ? StatusBaseButton.Type.Danger
                                                : StatusBaseButton.Type.Normal
                 enabled: checkBox.checked || root.isInvitationPending
+                Layout.fillWidth: true
```

Comment:
I think this is causing the button to fill the complete popup width instead of wrapping to the text size:

![Screenshot 2023-02-17 at 13 46 38](https://user-images.githubusercontent.com/97019400/219657117-ff0542ca-b216-4559-859d-8e792bf33953.png)

Reply:
Yeah... StatusBaseButton bug :/

Reply:
Reverted to the default behavior; but this is a more of a generic button issue; will deal with it separately